spec/support/usps_ipp_helper.rb

```diff
@@ -117,6 +117,27 @@ def request_in_progress_proofing_results_args
     }
   end
 
+  def stub_request_with_timeout_error(*_response)
```

Comment:
Couple thoughts:

- The other methods in this file are named specific to the endpoint they're stubbing, and I think we should follow that pattern here
- If we're not using the argument, do we need it? Many of the other methods here omit it, so I also don't see it being necessary purely out of consistency

```suggestion
  def stub_request_proofing_results_with_timeout_error
```